agent/test-agent-cli/src/test.rs

```diff
@@ -0,0 +1,29 @@
+use crate::error::Result;
+use crate::Client;
+use crate::{test_done, test_error, test_start};
+use argh::FromArgs;
+
+#[derive(Debug, FromArgs, PartialEq)]
+#[argh(subcommand, name = "test", description = "test state Done/start/Error")]
```

Comment:
nit: "done/start/error" could/should we do `test-done` etc instead or even `send-test-done` could also be `test-agent terminate`?